### ISSUE TITLE
[BUG] Accessibility - App - Onboarding video needs better screen reader labeling and navigation order of Play/Pause button

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#5108]
Describe the issue
_There is already a bug for the part where this video should not be automatically playing, ( https://github.com/rokwire/illinois-app/issues/4863 )._

 This bug is to address the fact that it needs to be navigable by assistive technology like screen readers and labels need to make sense as well as the navigation order. 

Success Criterion 2.4.7: Focus Visible (Level AA)
Goal: Ensure that users can see which element has keyboard focus.
Requirement: Any keyboard-operable user interface must have a mode where the keyboard focus indicator is visible.
Importance: Without a visible focus indicator, sighted keyboard users cannot effectively navigate the page.

Success Criterion 2.4.13: Focus Appearance (Level AAA)
Goal: Make the keyboard focus indicator more noticeable.
Requirement: The focus indicator must be of sufficient size and contrast. Specifically, it should be at least as large as a 2 CSS pixel thick perimeter of the unfocused component and have a contrast ratio of at least 3:1 between the focused and unfocused states.

**To Reproduce**
1. Make sure to clear cache to prompt Onboarding screens! 
2. Using TalkBack Go to 'IL APP'
3. Click on 'Continue' until Video screen
4. Screen reader focus lands on the back button first. 
5. Next, while video is autoplaying user hears "double tap to activate"

**Expected behavior**
The video should NOT be automatically playing. 
_(This is covered in Bug #4863. This button needs a white outline added to increase visibility as well #4873 . )_
The user focus should go from back button to a button labeled "Play/pause" allowing the user to start video should they choose to.  Screen reader user should hear "Play/pause" button. 

**Actual behavior**
The video auto plays, then the focus for play pause is over the entire page not the button itself. Screen reader user hears "double tap to activate" unsure of what they will be activating until doing so. 

**Screenshots**
Actual behavior:

![Image](https://github.com/user-attachments/assets/a502cfff-d664-4e52-ae9a-c47e38265891)

![Image](https://github.com/user-attachments/assets/9f7f1f95-8dfb-4689-8488-de9d50944486)

Attaching a recording of this happening and how confusing this is to a user without sight. 

https://github.com/user-attachments/assets/88323943-12fb-4d66-b395-9a721df198c1

**Smartphone (please complete the following information):**
 - Device: Google Pixel
 - OS: Android
 - Browser N/A this is an apk
 - Version 7.0.20 PROD
 
**Additional context**
